rotkehlchen/tests/data_migrations/test_migrations.py

```diff
@@ -47,6 +48,61 @@ def __init__(self, db) -> None:
         self.msg_aggregator = db.msg_aggregator
 
 
+def assert_progress_message(msg, step_num, description, migration_version, migration_steps) -> None:  # noqa: E501
+    assert msg['type'] == 'data_migration_status'
+    assert msg['data']['start_version'] == migration_version
+    assert msg['data']['target_version'] == LAST_DATA_MIGRATION
+    migration = msg['data']['current_migration']
+    assert migration['version'] == migration_version
+    assert migration['total_steps'] == (migration_steps if step_num != 0 else 0)
+    assert migration['current_step'] == step_num
+    if description is not None:
+        assert description in migration['description']
+    else:
+        assert migration['description'] is None
+
+
+def assert_ws_messages(
+        websocket_connection: 'WebsocketReader',
+        migration_steps: int,
+        chain_to_address: list[dict],
+        migration_version: int,
+) -> None:
+    num_messages = migration_steps + 1  # +1 for the message for migrated address
+    websocket_connection.wait_until_messages_num(num=num_messages, timeout=10)
+    assert websocket_connection.messages_num() == num_messages
+
+    for i in range(migration_steps):
+        msg = websocket_connection.pop_message()
+        if i == migration_steps:  # message for migrated address
```

Comment:
this assumes that this function is for migrations where we add addresses? maybe state it in the doc